linkerd/app/outbound/src/http/concrete.rs

```diff
@@ -163,7 +149,20 @@ impl<N> Outbound<N> {
                 .push_on_service(svc::MapErr::layer_boxed())
                 .lift_new_with_target()
                 .push(
-                    http::NewClassifyGateSet::<classify::Response, _, _, _>::layer_via(mk_breaker),
+                    http::NewClassifyGateSet::<classify::Response, _, _, _>::layer_via({
+                        // This sets the capacity of the channel used to send
+                        // response classifications to the failure accrual task.
+                        // Since the number of messages in this channel should
+                        // be roughly the same as the number of requests, size
+                        // it to the request queue capacity.
+                        // TODO(eliza): when queue capacity is configured
+                        // per-target, extract this from the target instead.
+                        let channel_capacity = config.http_request_queue.capacity;
```

Comment:
Don't we have a queue config in the balance target?

Reply:
lemme check, i thought we hadn't yet...

Reply:
took a look, the `queue::Capacity` param isn't wired up with any of the target types yet.